syntax = "proto3";

package rxsamplesmetadata;

message RxSamplesMetadata {
  uint32 sequence_num = 1;
  uint32 numberofreceivesamples = 2;
  double sequence_time = 3;
  double initialization_time = 4;
  double sequence_start_time = 5;
  uint64 ringbuffer_size = 6;
}
